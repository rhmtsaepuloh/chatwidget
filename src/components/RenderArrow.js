import React from "react";

const RenderArrow = ({ onClick, children }) => {
  return (
    <button
      className='h-12 w-12 rounded-full flex items-center justify-center'
      onClick={onClick}
      style={{ background: "rgba(0, 0, 0, 0.2)" }}>
      {children}
    </button>
  );
};

export default RenderArrow;
