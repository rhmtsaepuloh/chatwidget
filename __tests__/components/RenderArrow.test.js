import React from "react";
import { shallow } from "enzyme";
import RenderArrow from "../../src/components/RenderArrow";

const props = {
  onClick: jest.fn(),
  children: React.Component,
};

describe("render correctly render Arrow components", () => {
  test("renders", () => {
    const wrapper = shallow(<RenderArrow {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
