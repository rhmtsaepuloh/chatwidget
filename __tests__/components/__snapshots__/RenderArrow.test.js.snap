// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly render Arrow components renders 1`] = `ShallowWrapper {}`;
